ansible/vars/topo_t0-116.yml

```diff
@@ -87,8 +143,8 @@ configuration_properties:
     max_tor_subnet_number: 16
     tor_subnet_size: 64
     spine_asn: 65534
-    leaf_asn_start: 64600
-    tor_asn_start: 65100
+    leaf_asn_start: 4200064600
```

Comment:
are we using such long ASN number?

Reply:
Yes,  I am purposely testing 4-bytes ASN.